### DISCUSSION TITLE
Hide holiday

### DISCUSSION THREAD

**Discussion by andre0ani** [williamtroup/Calendar.js#260] [Q&A]
Hello,
Is it possible to hide some text like Chrimas Day, boxing day, etc ?
I tried with this but it's not working. 

        var calendarInstance1 = new calendarJs("calendar1", __TRANSLATION_OPTIONS, {
            exportEventsEnabled: true,
            showHolidays: false,
            views: {
                fullMonth: {
                    showTimesInEvents: false,
                }
            }
        });

And is it possible to hide some icons in the title bar, at left or right please ?

Thanks in advance,
Have a nice day,

Patrice

**Comment by william-troup** [comment]
Hi,

Take a look at the documentation here:
https://calendar-js.com/documentation/public-functions.html

You will see a public function called "addHolidays" and "removeHolidays".  Using these, you can add, or remove specific holidays from the display.  You can also set them in the options you use to initialize the Calendar.  You can view those options here:
https://calendar-js.com/documentation/options.html

As for the buttons in the title bar, take a look at the options.  You can disable all sorts of features.  However, if you want to just hide the buttons, this can be done using CSS.

Hope this helps.

William Troup

**Reply by andre0ani** [reply]
Thanks, I'll look into it.
Is the vacation name the one, case sensitive, that is displayed on the screen? Like Christmas Eve or Boxing Day, for example? It can be laborious if I want to remove all that.

Best regards,
Patrice

**Comment by andre0ani** [comment]
I see the option showHolidays with a boolean.
        var calendarInstance1 = new calendarJs("calendar1", __TRANSLATION_OPTIONS, {
            exportEventsEnabled: true,
            views: {
                fullMonth: {
                    showTimesInEvents: false,
                }
            }
        });
        calendarInstance1.showHolydays: false,
